src/module/item/physical/index.ts

```diff
@@ -234,13 +234,15 @@ abstract class PhysicalItemPF2e extends ItemPF2e {
         super.prepareDerivedData();
 
         const systemData = this.system;
-        systemData.identification.identified = {
-            name: this.name,
-            img: this.img,
-            data: {
-                description: { value: this.description },
-            },
-        };
+        if (!systemData.identification.identified) {
+            systemData.identification.identified = {
+                name: this.name,
+                img: this.img,
+                data: {
+                    description: { value: this.description },
+                },
+            };
+        }
```

Comment:
This is due to a core bug in which data preparation is done twice.

```suggestion
        systemData.identification.identified ??= {
            name: this.name,
            img: this.img,
            data: {
                description: { value: this.description },
            },
        };
```